CHANGES

```diff
@@ -1,12 +1,16 @@
 pint-pandas Changelog
 =====================
 
-0.6 (unreleased)
+0.6 (2024-06-16)
 ----------------
 
 - Fix dequantify duplicate column failure #202
 - Fix astype issue #196
 - Support for `__array_ufunc__` and unary ops. #160
+- Support for NumPy verson 2
+- Minimum Pint version is 0.24
+- Minimum Pandas version is 2.2
```

Comment:
These are not the versions currently specified in `pyproject.toml`:
https://github.com/hgrecco/pint-pandas/blob/0e6a5ac71ec0dd009a8be9d56b53142eadab8ca8/pyproject.toml#L25-L30